src/main/java/com/rocketden/main/service/ProblemService.java

```diff
@@ -164,6 +242,38 @@ public ProblemTestCaseDto createTestCase(String problemId, CreateTestCaseRequest
         return ProblemMapper.toTestCaseDto(testCase);
     }
 
+    // Check to make sure test case inputs and outputs are Gson-parsable
+    protected void validateGsonParseable(String input, List<ProblemInputDto> types) {
```

Comment:
What was your thinking behind making this `protected`?

Also, nit: Change to `validateInputsGsonParseable` or something similar, to differentiate it from the below method. (Not a huge deal either way, but I think it could be helpful to make it a different method name so as to avoid confusion.)

Reply:
Mainly so that the method could directly be tested (not sure if there's a way to do that without using something new like Powermock)

Also sounds good on the nit